automl/google/cloud/automl_v1beta1/proto/tables_pb2.py

```diff
@@ -694,6 +758,18 @@
   
   
   Attributes:
+      additional_optimization_objective_config:
+          Additional optimization objective configuration. Required for
+          ``MAXIMIZE_PRECISION_AT_RECALL`` and
+          ``MAXIMIZE_RECALL_AT_PRECISION``, otherwise unused.
+      optimization_objective_recall_value:
+          Required when optimization\_objective is
+          "MAXIMIZE\_PRECISION\_AT\_RECALL". Must be between 0 and 1,
+          inclusive.
+      optimization_objective_precision_value:
+          Required when optimization\_objective is
+          "MAXIMIZE\_RECALL\_AT\_PRECISION". Must be between 0 and 1,
+          inclusive.
```

Comment:
Add `additional_optimization_objective_config`, `optimization_objective_recall_value`, `optimization_objective_precision_value` to `TablesModelMetadata`